addons/test_mail_full/models/test_mail_models.py

```diff
@@ -57,3 +57,17 @@ class MailTestSMSSoLike(models.Model):
 
     name = fields.Char()
     partner_id = fields.Many2one('res.partner', 'Customer')
+
+
+class MailTestPortal(models.Model):
```

Comment:
Could be defined before mail.test.sms (just to keep an alphabetical ordering when possible)

Reply:
Done :+1: